### PR TITLE
Margin 8px

### PR DESCRIPTION
**Pull request by SahilRw**

### CODE REVIEW

File: src/pages/LandingPage/EmailConfirm.jsx [hunk 1]
```diff
@@ -8,12 +8,12 @@ const EmailConfirm = ({email, message}) => {
         <br />
         <div className='bg-gray-800 p-6 rounded-lg'>
             <h3 className="text-xl font-bold mb-4">Here is a copy of your email: </h3>
-            <p className='py-1'>
+            <p className='my-2'>
```
Comment (BjornGjorv) [inline src/pages/LandingPage/EmailConfirm.jsx:11]: my-2 sets the padding in the Y-direction, meaning both top and bottom. In this case the requirement was 8px bottom, so then mb-2 (margin-bottom-2) should be used. 

But, both lookes fine, so its not really a problem right now. However, many designers are strict and are salty with devs that doesnt pay attention, hehe. So its something to think about!

File: src/pages/LandingPage/EmailConfirm.jsx [hunk 2]
```diff
@@ -8,12 +8,12 @@ const EmailConfirm = ({email, message}) => {
...
                 <span className="font-bold">Email Address:</span> {email}
             </p>
             <p>
-                <span className="font-bold">Message:</span>
-                    <div className='h-0.5 bg-white'></div>  
+                    <span className="font-bold">Message:</span>
+                    <div className='h-0.5 bg-white mb-2'></div>
```
Comment (BjornGjorv) [inline src/pages/LandingPage/EmailConfirm.jsx:16]: nice! 👌

**Review by BjornGjorv — COMMENTED** [review]
added some comments

File: src/pages/LandingPage/ContactForm.jsx [hunk 3]
```diff
@@ -13,7 +13,6 @@ const ContactForm = () => {
   }, [email, message]);
 
   if (state.succeeded) {
-    // TODO: Extract this into a seperate card component with some nice styling
```
Comment (BjornGjorv) [inline src/pages/LandingPage/ContactForm.jsx:16]: well spotted 👍 👍 👍

File: src/pages/LandingPage/EmailConfirm.jsx (hunk 1 above)
Comment (SahilRw) [reply src/pages/LandingPage/EmailConfirm.jsx:11]: Okay. I will work on that.

**Comment by BjornGjorv** [comment]
Handeled in https://github.com/SahilRw/task-tracker/pull/16. Can be closed

**Pull request closed without merging**